taichi/util/bit.h

```diff
@@ -47,6 +47,21 @@ TI_FORCE_INLINE float32 half_to_float(const uint16 x) {
                              0x007FE000)));  // sign : normalized : denormalized
 }
 
+TI_FORCE_INLINE uint16 float_to_half(const float32 x) {
```

Comment:
We have a F16 library submodule in the externals folder. Use that pls for IEEE correctness and hardware intrinsics on ARM CPUs